### PR TITLE
MOVEit Transfer vulnerability

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
A new pack which handles MOVEit Transfer CVE-2023-34362 vulnerability.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.8.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml [hunk 1]
```diff
@@ -0,0 +1,2742 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: 9d3492f4-1d99-444a-8ed1-d8eac62cb083
+    type: playbook
+    task:
+      id: 9d3492f4-1d99-444a-8ed1-d8eac62cb083
+      version: -1
+      name: QRadarFullSearch
```
Comment (altmannyarden) [inline Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:822]: need to change the names of Qradar :)  (few tasks)

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml [hunk 2]
```diff
@@ -0,0 +1,2742 @@
...
+      {
+        "position": {
+          "x": -550,
+          "y": 1730
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (altmannyarden) [inline Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:858]: should be true

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml [hunk 3]
```diff
@@ -0,0 +1,2742 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: df110072-3316-4eb8-8e68-7325f8dc9e4a
+    type: playbook
+    task:
+      id: df110072-3316-4eb8-8e68-7325f8dc9e4a
+      version: -1
+      name: Block Indicators - Generic v3
```
Comment (altmannyarden) [inline Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:948]: the default values for blocking is based on DBOTscore - should we change them? to the ones we extracted?

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml [hunk 4]
```diff
@@ -0,0 +1,2742 @@
...
+      {
+        "position": {
+          "x": 1040,
+          "y": 1550
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (altmannyarden) [inline Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:1980]: should be true

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml [hunk 5]
```diff
@@ -0,0 +1,2742 @@
...
+      {
+        "position": {
+          "x": -330,
+          "y": 2390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (altmannyarden) [inline Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:2669]: should be true

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml (hunk 4 above)
Comment (melamedbn) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:1770]: Done

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml (hunk 3 above)
Comment (melamedbn) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:890]: Done

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml (hunk 1 above)
Comment (melamedbn) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml]: Done

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml (hunk 2 above)
Comment (melamedbn) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml]: Done

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml (hunk 5 above)
Comment (melamedbn) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:2459]: Done

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml [hunk 6]
```diff
@@ -984,14 +926,6 @@ tasks:
...
                   simple: file
-          - - operator: greaterThanOrEqual
-              left:
-                value:
-                  simple: DBotScore.Score
-                iscontext: true
-              right:
-                value:
-                  simple: "3"
           accessor: Indicator
```
Comment (altmannyarden) [inline Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:929]: there is no key for file.indicator.

should be file.sha254

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml [hunk 7]
```diff
@@ -1008,58 +942,11 @@ tasks:
...
               ignorecase: true
-          - - operator: greaterThanOrEqual
-              left:
-                value:
-                  simple: DBotScore.Score
-                iscontext: true
-              right:
-                value:
-                  simple: "3"
           accessor: Indicator
```
Comment (altmannyarden) [inline Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:945]: same here, IP.address

Comment (altmannyarden) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:945]: or from the extracted context key

Comment (altmannyarden) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:945]: its depend on if we run enrichment

Comment (melamedbn) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:945]: Done

File: Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml (hunk 6 above)
Comment (melamedbn) [reply Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml:929]: Done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**